irakozetony/my-brand
# #PodhBJJy implement home and about page

#### what does this PR do?
this PR implements the home and about page

#### description of the task to be completed
- create the home page
- create the about page

#### how should this be manually tested
1. clone the [repo](https://github.com/irakozetony/my-brand/)
2. checkout branch feat-home-and-about-page-PodhBJJy
3. check the index.html and about.html

#### what are the relevant pivotal tracker stories
https://trello.com/c/PodhBJJy/12-implement-home-and-about-page
